merkle_tree/Cargo.toml

```diff
@@ -24,7 +24,7 @@ displaydoc = { workspace = true }
 hashbrown = { workspace = true }
 hex = "0.4.3"
 itertools = { workspace = true, features = ["use_alloc"] }
-jf-poseidon2 = { path = "../poseidon2" }
+jf-poseidon2 = { version = "0.1.0", git = "https://github.com/EspressoSystems/jellyfish" }
```

Comment:
should we also have a version number bump for jf-merkle-tree after CRHF is done?

Reply:
yes, I'll introduce the CRHF in the next PR, then another one after that to bump jf-merkle-tree